tests/integration.rs

```diff
@@ -70,6 +70,10 @@ mod tests {
     const FOCAL_IMAGE_NAME_VHD: &str = "focal-server-cloudimg-arm64-custom.vhd";
     #[cfg(target_arch = "x86_64")]
     const FOCAL_IMAGE_NAME_VHD: &str = "focal-server-cloudimg-amd64-custom-20210609-0.vhd";
+    #[cfg(target_arch = "aarch64")]
+    const FOCAL_IMAGE_NAME_VHDx: &str = "focal-server-cloudimg-arm64-custom.vhdx";
+    #[cfg(target_arch = "x86_64")]
+    const FOCAL_IMAGE_NAME_VHDx: &str = "focal-server-cloudimg-amd64-custom-20210609-0.vhdx";
```

Comment:
From the log [here](https://github.com/cloud-hypervisor/cloud-hypervisor/pull/2888/checks?check_run_id=3128021107), you see the error is from the `Clippy (integration tests)` step with the following command from the GH workflow, and you can rerun the same command on your dev machine to reproduce the error:

```
Run cargo clippy --all --all-targets --tests --features "integration_tests" -- -D warnings
  cargo clippy --all --all-targets --tests --features "integration_tests" -- -D warnings
```

Also, for huge changes, I normally do a PR to my own fork to test those GH actions (which will avoid running the complete CI). Of course, you can manually do all of them by following the workflow defined in the repo, say [here](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/.github/workflows/quality.yaml#L33-L61).

Reply:
Thanks for the tips. It will help make a successful PR.